### ISSUE TITLE
Selection meshes are too large

### ISSUE THREAD

**Issue by vrad-exe** [BEEmod/BEEMOD#5]
The selection meshes of all the BEE items are slightly larger than the visible models. This has been fixed in the BEE2.4 versions of most of the models, so it should be possible to just backport them from there.

**Comment by LautaroL20** [comment]
Yeah sure thing, but isn´t this version somewhat dead? I would not be so sure if the efforts to backporting 2.4´s meshes to this abandoned and unknown version would worth for someone...

**Comment by BenVlodgi** [comment]
@Luke18033 Is this true for every selection mesh?

**Comment by vrad-exe** [comment]
Yeah
